metadata_mapper/mappers/mapper.py

```diff
@@ -11,7 +11,8 @@
 from datetime import date
 
 import settings
-from .validator import Validator, ValidationErrors
+from validator.validator import Validator
+from validator.validation_log import ValidationLog
```

Comment:
Similarly, change this to:

`from validator import validation_log` 

or even collapse the 2 lines into:

` from validator import validator, validation_log`

Reply:
Then when you instantiate the classes from these modules, like on line 89, you can do:

`validator = validator.Validator`

Reply:
Hmm, I do like using classes without having to specify a module prefix, especially when it's not at all confusing where the class came from, as in this case.

Reply:
Hmm yeah I see what you're saying. Well, maybe we can just leave this as is for now and circle back when we choose a workflow management tool and need to figure out the best approach to packaging up rikolti.